lib/fetchers/fetcher.py

```diff
@@ -18,12 +16,16 @@
 from dplaingestion.selector import setprop
 from dplaingestion.selector import getprop as get_prop
 from dplaingestion.utilities import iterify, couch_id_builder
+import requests
+import re
```

Comment:
Argh, how annoying, this old code. I am observing how `file_fetcher` does `from dplaingestion.fetchers.fetcher import *` and then `edan_fetcher` does `from dplaingestion.fetchers.file_fetcher import *`.  This is a serious Python anti-pattern, though -- it really makes me feel unwell. This leads to all kinds of head-scratching down the line when imports are changed in the upstream modules or we're trying to analyze the code downstream.

It is outside of the scope of the relevant ticket to refactor the fetchers, and I know we don't want to spend too much extra time on Ingestion 1, but I think it's a quality of life issue, and I'll try to submit a small PR some time to fix this problem.